src/UnitTests/IMappingExpression/NonGenericProjectEnumTest.cs

```diff
@@ -73,7 +73,7 @@ public void ProjectingEnumToString()
         {
             var customers = new[] { new Customer() { FirstName = "Bill", LastName = "White", CustomerType = CustomerType.Vip } }.AsQueryable();
 
-            var projected = _mapper.Map<CustomerDto[]>(customers);
+            var projected = customers.ProjectTo<CustomerDto>(_mapper.ConfigurationProvider);
```

Comment:
Is that some kind of inception thing?

On Monday, May 16, 2016, Tyler Carlson <notifications@github.com> wrote:

> In src/UnitTests/IMappingExpression/NonGenericProjectEnumTest.cs
> <https://github.com/AutoMapper/AutoMapper/pull/1275#discussion_r63368748>:
>
> > @@ -73,7 +73,7 @@ public void ProjectingEnumToString()
> >          {
> >              var customers = new[] { new Customer() { FirstName = "Bill", LastName = "White", CustomerType = CustomerType.Vip } }.AsQueryable();
> >
> > -            var projected = _mapper.Map<CustomerDto[]>(customers);
> > +            var projected = customers.ProjectTo<CustomerDto>(_mapper.ConfigurationProvider);
>
> OK then it's official. AM doesn't support projection when mapping
> IQueryable lol
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/AutoMapper/AutoMapper/pull/1275/files/346c230e18f74d8df735efe3ec820c4b58da6251..60d7f3c64cd3483339a85e9e6fd84a6605d96b4b#r63368748>
>